### PR TITLE
added extension for popToRootViewController

### PR DESCRIPTION
**Pull request by vilapuigvila**
## Checklist

- [x] New Extension
- [ ] New Test
- [ ] Changed more than one extension, but all changes are related
- [ ] Trivial change (doesn't require changelog)

### CODE REVIEW

**Comment by lfarah** [comment]
Hey @vilapuigvila, thanks for contributing!
Please add tests and documentation like this:
``` swift
/// EZSE: Goes back to root View Controller
public func popToRootVC() {
         _ = navigationController?.popToRootViewController(animated: true)
}
```

File: Sources/UIViewControllerExtensions.swift [hunk 1]
```diff
@@ -224,7 +224,12 @@ extension UIViewController {
     public func popVC() {
         _ = navigationController?.popViewController(animated: true)
     }
-
+    
+    /// EZSwiftExtensions
+    public func popToRootVC() {
+        _ = navigationController?.popToRootViewController(animated: true)
```
Comment (piv199) [inline Sources/UIViewControllerExtensions.swift]: why do we suppress returning popped view controllers?

Comment (lfarah) [inline Sources/UIViewControllerExtensions.swift]: I personally never use this values, so I think it's ok.

@piv199 how do you use this returned value?

Comment (piv199) [inline Sources/UIViewControllerExtensions.swift]: @lfarah I'm not using them :)

**Comment by vilapuigvila** [comment]
who use this values 😉

**Comment by lfarah** [comment]
Hey @vilapuigvila, almost everything perfect!
Now I just need you to remove [this commit](https://github.com/goktugyil/EZSwiftExtensions/pull/330/commits/deeda0e0e2b1674b76d0d9f33a8392b3f62b3014) and open it in another PR

**Comment by lfarah** [comment]
@Khalian we're good over here?

**Comment by Khalian** [comment]
No I mean't squashing commits not reverting. 

https://egghead.io/lessons/javascript-how-to-squash-multiple-git-commits

**Comment by vilapuigvila** [comment]
ok let me take a look about that, thanks again.

**Comment by Khalian** [comment]
Squashed PR here : https://github.com/goktugyil/EZSwiftExtensions/pull/341

**Pull request closed without merging**